testsuite/features/reposync/srv_sync_repo_sync.feature

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Strict Mode for Channel Synchronization
+  In order to control strict mode synchronization
+  As an admin user
+  I want to be able to toggle strict mode synchronization via the UI and command line
+
+Scenario: Create Test-strict-channel
+  Given I am authorized for the "Admin" section
+    And I prepare a channel clone for strict mode testing
```

Comment:
I'm not sure about this step, it hides too much actions, and it's too specific for this test case.

Reply:
yes, i thought so but i don't know how / where else to prepare the modified repository which is needed for the test, is there a script that would be run before cucumber execution that i could outsource this preparation step to ?

Reply:
No worry for now, maybe we can come back to this in the future.
You have already enough stuff to worry in your PR :D